source/_drafts/Video-js-7-4.md

```diff
@@ -0,0 +1,227 @@
+  - languages
+  - live-ui
+  - focus-visible
+---
+
+It's time to have an overview of Video.js 7.4, first released early December.
+The big new feature for this release is a UI that allows you to seek during live streams.
+We updated focus-visible to work with our Menus, added more translations, added a replay option to the Play/Pause button, and many many fixes, multiple of which are accessibility related.
+
+We also dropped our usage of Grunt from our build process.
+We owe a lot to grunt as it has served us well but it was time to move on.
+
+Video.js 7.4.1 is currently the latest release with 7.4.2 out as a pre-release until next week.
+
+## Thanks
+
+Before continuing, I'd like to thank everyone that was involved, we really appreciate your contributions!
+There were a total of 14 first time contributors, I think this is a historic high for us and I hope this continues!
+
+- [@BrandonOCasey](https://github.com/BrandonOCasey)
+- [@gesinger](https://github.com/gesinger)
+- [@gjanblaszczyk](https://github.com/gjanblaszczyk) (first time contributor!)
+- [@valse](https://github.com/valse) (first time contributor!)
+- [@gstrat88](https://github.com/gstrat88)
+- [@eranshmil](https://github.com/eranshmil) (first time contributor!)
+- [@carlmorris](https://github.com/carlmorris) (first time contributor!)
+- [@bartlomein](https://github.com/bartlomein) (first time contributor!)
+- [@DanielRuf](https://github.com/DanielRuf) (first time contributor!)
+- [@Quenty31](https://github.com/Quenty31) (first time contributor!)
+- [@fketchakeu](https://github.com/fketchakeu) (first time contributor!)
+- [@OwenEdwards](https://github.com/OwenEdwards)
+- [@alex-barstow](https://github.com/alex-barstow)
+- [@vitaliytv](https://github.com/vitaliytv) (first time contributor!)
+- [@oaprograms](https://github.com/oaprograms) (first time contributor!)
+- [@xjoaoalvesx](https://github.com/xjoaoalvesx) (first time contributor!)
+- [@webdeveloperpr](https://github.com/webdeveloperpr) (first time contributor!)
+- [@smbea](https://github.com/smbea) (first time contributor!)
+- [@dustin71728](https://github.com/dustin71728) (first time contributor!)
+
+## Live UI
+Video.js has supported live streams for a while, either natively, or via [videojs-http-streaming (VHS)](https://github.com/videojs/http-streaming).
+However, this UI was very minimal, it disabled the progress bar and basically just allowed paused, though, there was an indicator that this was a live stream.
+
+![Video.js playing a live stream](live ui.png)
+
+The new UI, looks pretty similar to the previous live UI and the regular control bar.
+The "live" indicator moves to the right side of the progress bar and also indicates whether we are playing back live or we are behind live.
+Clicking this button will seek to the "live point".
+The Time Displays and Tooltips will show time with respect to the live point, so, current time at the live point will be `0` and if you seek back 30 seconds, it'll show `-00:30`.
```

Comment:
Should "Time Displays" and "Tooltips" be capitalized? Doesn't seem like they are proper nouns.

Also, `, so,` should be `; so,`. Don't think you need the comma before `it'll`.